src/styles/typography/index.md.njk

```diff
@@ -99,6 +99,27 @@ Use the `govuk-link--no-visited-state` modifier class where it is not helpful to
 
 {{ example({group: "styles", item: "typography", example: "link-no-visited-state", html: true, open: true}) }}
 
+If it's an external link to a non-government website, make that clear in the link text. For example, 'read advice on writing link text from [name of organisation]'. There's no need to say explicitly that you're linking to an external site. [Do not use an external link icon](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/).
+
+### Opening links in a new tab
+
+Avoid opening links in a new tab or window. It can be disorienting - and [can cause accessibility problems for people who aren't able to perceive the link visually](https://www.w3.org/TR/WCAG20-TECHS/G200.html).
```

Comment:
I don't think saying "people who aren't able to perceive the link visually" is accurate. It would be more accurate to say "people who aren't able to perceive the new tab visually". The main confusion comes from not knowing that a new tab has opened, which has nothing to do with the visuals of the link. But it's also for people with cognitive issues, not just blind people.

Reply:
Thanks @selfthinker - we'll change the text so it says `can cause accessibility problems for people who aren't able to visually perceive that the new tab has opened`